File: Auto-generated-tasks/Difficulty-1/Task-2/Solution.f90
integer:: N = 6, i
character (len=6), dimension(N) :: Spisok
do i = 0, N/3
	Spisok(i * 3) = "No"
	Spisok(i * 3 + 1) = "Buf517"
	Spisok(i * 3 + 2) = "K776"
end do
do i = 0, N
	print *, "Spisok[", i, "]=", Spisok(i)
end do

File: Auto-generated-tasks/Difficulty-1/Task-2/Solution.pas
Var
	N:=6, i: integer;
	Spisok:Array[0..N] of string;
Begin
	for i:= 0 to N/3 do begin
		Spisok[i * 3] := "No";
		Spisok[i * 3 + 1] := "Buf517";
		Spisok[i * 3 + 2] := "K776";
	end;
	for i:= 0 to N do begin
		WriteLn("Spisok[",i,"]=",Spisok[i]);
	end;
End.